guides/common/modules/proc_configuring-repositories-el.adoc

```diff
@@ -23,6 +23,16 @@ ifdef::katello[]
 ----
 # {package-manager} localinstall https://yum.theforeman.org/katello/{KatelloVersion}/katello/el{distribution-major-version}/x86_64/katello-repos-latest.rpm
 ----
+
+ifeval::["{distribution-major-version}" == "8"]
++
+. Install the `centos-release-ansible-29` package to enable ansible collection support
```

Comment:
"enable ansible collection support" sounds like this repo contains it, which is not correct.
```suggestion
. Install the `centos-release-ansible-29` package to enable repositories for dependencies of the Ansible collection support
```

Reply:
I was too quick. I will fix this now sorry @evgeni

Reply:
@jlsherrill is ansible enabled by default on 4.1, or optional? If enabled by default, I'd not explain *why* it's needed at all. We don't for the other repos.